k8s-cluster/README.md

```diff
@@ -321,13 +307,39 @@ TLS connection with Postgres utilizes the secret generator at the end of the [ku
 
 - the files can be store any were and referenced inthe code with the full path.
 
-**⚠️ NOTE**: TLS connection for MongoDB is still in progress
+> ⚠️ **NOTE:** TLS connection for MongoDB is still in progress
+
+### 4.4 Buckets
 
-### 4.4 Enviroment without buckets
+To configure ZHE to store uploaded images and files as object in buckets all this varibales need to provided in kustomization.yaml
 
-**⚠️ NOTE**: This option will only be avalible during the `BETA` phase
+```yaml
+configMap:
+  - bucket_access_key_id=<access_key_id>
+  - bucket_region=<bucket_region>
+  - bucket_domain=<bucket_public_domain>
+  - files_bucket_name=<bucket_name>
+  - images_bucket_name=<bucket_name>
+secret:
+  - bucket_secret_access_key=<some-key>
+```
 
-**⚠️ NOTE**: Using this option the enviroment is fucntional but files can **not** be uploaded. A error message will be shown.
+> ⚠️ **NOTE:** At the moment only AWS S3 API is supported.
+> ⚠️ **NOTE:** At the moment only IAM credentials are supported. There is some work in progress to support roles.
+
+IAM credetials are used by ZHE to write ( `put` ) objects and to create temporary pre-signed links.
```

Comment:
There's a lot of duplication between this section ("4.4 Buckets") and "2.6 Files and Images Storage" above. We should combine them together. I think we can just move all the things in 4.4 to 2.6.

Reply:
keeping both for now.. 
2.6 is explanation about the requirements and usage
4.4 instruction to setup and use